playground/index.js

```diff
@@ -1,6 +1,12 @@
 const { jsPDF } = window.jspdf
 const DOMPurify = window.DOMPurify
 
+DOMPurify.addHook('afterSanitizeAttributes', node => {
+  if(node.hasAttribute('xlink:href') && !node.getAttribute('xlink:href').match(/^#/)) {
```

Comment:
We should also check the `href` attribute without the `xlink` namespace.

Reply:
Good point, I didn't know that it is deprecated in SVG 2. I have amended the commit and also added `playground` to prettier check.